packages/docs/versioned_docs/version-2.0.0-alpha-3/testnet-deployment.md

```diff
@@ -35,3 +35,7 @@ This URL can be directly used in a [network prover](circuits-api/network-prover)
 :::danger
 These keys have not had a secure phase 2 trusted setup. **Use these keys at your own risk.**
 :::
+
+TODO: explain this warning
+
+Read more about trusted setups [here](https://vitalik.ca/general/2022/03/14/trustedsetup.html)
```

Comment:
Is there any policy / guideline for providing links to outside sources?
Personally I would appreciate this supporting context being made easily available to me, but maybe this is not the appropriate place?

Reply:
I think it's okay to link to outbound sites for things like this.